### ISSUE TITLE
The Simplified OptiKey #accessibility

### ISSUE THREAD

**Issue by alexandre-mbm** [JuliusSweetland/OptiKey#860]
Hello, guys!

I am working an accessiblity approach for my father, a ALS person in advanced state (issue #855). Initially is an simplified OptiKey with two screens only. The first "point of use" is a dynamic keyboard maximised (ref.: #857) always.

![01-Main](https://user-images.githubusercontent.com/957501/188770316-869baa75-4e45-447c-8c78-5c5d1df954ae.jpg)

Another screen is the dynamic keyboards selector as a "final point" for the user. From it the user can only "back" to maximised dynamic keyboards.

![02-Selector](https://user-images.githubusercontent.com/957501/188770500-b40082db-e526-4a8c-9af8-f0055c515864.jpg)

See the changes for this workaround here: https://github.com/OptiKey/OptiKey/compare/v3.2.3...ENE:OptiKey:alberto

## The problem

The initial change on "selector" is working as [in this commit](https://github.com/ENE/OptiKey/commit/0705b4d8f83e7d2e4dcc993b59b6dfdf30a02e12) and shown in the image above. **BUT** the changes bellows are buggy:
```csharp
// Calibrate key
{
    Key newKey = new Key();
    newKey.SymbolGeometry = (Geometry)this.Resources["CalibrateIcon"];
    newKey.Text = JuliusSweetland.OptiKey.Properties.Resources.RE_CALIBRATE;
    newKey.Value = KeyValues.CalibrateKey;
    this.AddKey(newKey, this.mRows - 1, 1);
}

// Yes/No keyboard key
{
    Key newKey = new Key();
    newKey.SymbolGeometry = (Geometry)this.Resources["YesNoIcon"];
    newKey.Text = JuliusSweetland.OptiKey.Properties.Resources.YESNO;
    newKey.Value = KeyValues.ConversationConfirmKeyboardKey;
    this.AddKey(newKey, this.mRows - 1, 2);
}
```
Mentions: @kmcnaught, @AmaralVini, @alandbm

**Comment by AdamRoden** [comment]
@alexandre-mbm 
Is it possible that you are adding the two new keys in the keyboard's initial definition section of code, but afterward they are getting overlayed/replaced by blank keys?

**Comment by alexandre-mbm** [comment]
Yes, it **can be...** in tests: "blank buttons with actions, and no animations", but I looked for this on code already, a rebinding of objects, and I didn't see it.

**Comment by kmcnaught** [comment]
Hi @alexandre-mbm if I've understood correctly, you are trying to add a YesNo key and a Calibrate key in the bottom row. I think they are being replaced by empty keys in this block of code:

```
            // Add keyboard keys, or blanks
            int maxKeyboardsPerPage = (this.mCols - 2) * this.mRows;
            int totalNumKeyboards = folder.keyboards.Count;
            int remainingKeyboards = totalNumKeyboards - maxKeyboardsPerPage * pageIndex;
            int nKBs = Math.Min(remainingKeyboards, maxKeyboardsPerPage);
            int firstKB = maxKeyboardsPerPage * pageIndex;
```

I imagine this will be fixed by changing the first line to 
```
            int maxKeyboardsPerPage = (this.mCols - 2) * (this.mRows - 1);
```

Alternatively, you could comment out this line so no blank keys are added at all:
```
            // Add empty/inactive key for consistent aesthetic
            this.AddKey(new Key(), r, c);
```
